tests/modules/seqtk/rename/test.yml

```diff
@@ -1,21 +1,21 @@
 - name: seqtk rename test_seqtk_rename
   command: nextflow run tests/modules/seqtk/rename -entry test_seqtk_rename -c tests/config/nextflow.config
   tags:
-    - seqtk/rename
     - seqtk
+    - seqtk/rename
   files:
-    - path: /scicomp/home-pure/ngr8/tmp/tmp048xy0nm/seqtk/test.renamed.fasta.gz
+    - path: /scicomp/home-pure/ngr8/tmp/tmppxnl1so8/seqtk/test.renamed.fasta.gz
```

Comment:
I think this is the problem with the CI tests, try replacing `/scicomp/home-pure/ngr8/tmp/tmppxnl1so8` with `output/..`

Reply:
Changed the local tmp path `/scicomp/home-pure/ngr8/tmp/tmppxnl1so8` to `output`